src/KeywordSearch.jl

```diff
@@ -14,10 +14,13 @@ A list of replacements to automatically perform when preprocessing a [`Document`
 For example, if `KeywordSearch.AUTOMATIC_REPLACEMENTS == ["a" => "b"]`, then
 `Document("abc").text == "bbc"` instead of "abc".
 
-By default, `AUTOMATIC_REPLACEMENTS` contains only one replacement,
+By default, `AUTOMATIC_REPLACEMENTS` contains only one replacement:
+
+```jldoctest
+julia> KeywordSearch.AUTOMATIC_REPLACEMENTS
+1-element Vector{Pair{Union{Regex, String}, String}}:
+ r"[.!?><\-\n\r\v\t\f\s]+" => " "
```

Comment:
```suggestion
 r"[.!?><\-\v\f\s]+" => " "
```

Reply:
Reviewdog spotted this too, but you got there first! https://github.com/beacon-biosignals/KeywordSearch.jl/pull/23#discussion_r748609289